config/coordinator.sample.json

```diff
@@ -13,34 +13,51 @@
     "UpdateIntervalMs": 2000,
     "Queries": [
       {
-        "Description": "Average request duration (old)",
+        "Description": "Average server request duration (old)",
         "Query": "rate(mattermost_http_request_duration_seconds_sum[1m])/rate(mattermost_http_request_duration_seconds_count[1m])",
         "Threshold": 0.1,
         "Alert": true
       },
       {
-        "Description": "Average request duration",
+        "Description": "Average server request duration",
```

Comment:
Almost thought of removing these. I can't picture a case where they won't get covered by the client ones.

Reply:
Agree. We can remove, but let's keep the graph in the dashboard.

Reply:
Sure, this is just on the coordinator's side. We obviously want the whole picture on Grafana.